ci-operator/step-registry/sandboxed-containers-operator/peerpods/param-cm/sandboxed-containers-operator-peerpods-param-cm-commands.sh

```diff
@@ -12,6 +12,62 @@ cd /tmp || exit 1
 # can reference it.
 PP_CONFIGM_PATH="${SHARED_DIR:-$(pwd)}/peerpods-param-cm.yaml"
 
+handle_aws() {
+    local AWS_REGION
+    local AWS_SG_IDS
+    local AWS_SUBNET_ID
+    local AWS_VPC_ID
+    local INSTANCE_ID
+
+    oc -n kube-system get secret aws-creds -o json > aws-creds.json
+
+    AWS_ACCESS_KEY_ID="$(jq -r .aws_access_key_id aws-creds.json | base64 -d)"
+    export AWS_ACCESS_KEY_ID
+    AWS_SECRET_ACCESS_KEY="$(jq -r .aws_secret_access_key aws-creds.json | base64 -d)"
+    export AWS_SECRET_ACCESS_KEY
+
+    # TODO: remove me
+
+    ls -la ${CLUSTER_PROFILE_DIR} || true
+    cat ${CLUSTER_PROFILE_DIR}/.awscred || true
+
+    oc create secret generic peerpods-param-secret --from-file="${CLUSTER_PROFILE_DIR}/.awscred" -n default
+
+    INSTANCE_ID=$(oc get nodes -l 'node-role.kubernetes.io/worker' -o jsonpath='{.items[0].spec.providerID}' | sed 's#[^ ]*/##g')
+    AWS_REGION=$(oc get infrastructure/cluster -o jsonpath='{.status.platformStatus.aws.region}')
+    AWS_SUBNET_ID=$(aws ec2 describe-instances --instance-ids "${INSTANCE_ID}" --query 'Reservations[*].Instances[*].SubnetId' --region "${AWS_REGION}" --output text)
+    AWS_VPC_ID=$(aws ec2 describe-instances --instance-ids "${INSTANCE_ID}" --query 'Reservations[*].Instances[*].VpcId' --region "${AWS_REGION}" --output text)
+    AWS_SG_IDS=$(aws ec2 describe-instances --instance-ids "${INSTANCE_ID}" --query 'Reservations[*].Instances[*].SecurityGroups[*].GroupId' --region "${AWS_REGION}" --output text | tr ' \t' ',')
+
+    # Opening ports
+    for AWS_SG_ID in ${AWS_SG_IDS/,/ }; do
+        aws ec2 authorize-security-group-ingress \
+            --group-id "${AWS_SG_ID}" --protocol tcp --port 15150 \
+            --source-group "${AWS_SG_ID}" --region "${AWS_REGION}"
+        aws ec2 authorize-security-group-ingress \
+            --group-id "${AWS_SG_ID}" --protocol tcp --port 9000 \
+            --source-group "${AWS_SG_ID}" --region "${AWS_REGION}"
+    done
+
+    cat <<-EOF > "${PP_CONFIGM_PATH}"
```

Comment:
Hi, you may want to consider utilizing https://github.com/openshift/sandboxed-containers-operator/blob/devel/scripts/cm-helpers/pp-cm-helper.sh for that

Reply:
Hi @snir911 ! Thanks for your review!

Currently the test tool gets a configmap which looks like peer-pods-cm but it is not. However, we plan to move the logic from the test tool to the job in Prow and create the peer-pods-cm directly, then we will be able to use the pp-cm-helper.sh. Thanks for remind me that this script exists :)